docs/plugin-syntax-dynamic-import.md

```diff
@@ -4,6 +4,17 @@ title: @babel/plugin-syntax-dynamic-import
 sidebar_label: syntax-dynamic-import
 ---
 
+`@babel/plugin-syntax-dynamic-import` is needed to enable support for parsing `import()`
+
+This plugin is enabled by default since Babel 7.8.0, so you shouldn't need to enable this plugin separately.
+
+**Usage notes:**
+
+1. If you are using `@babel/preset-env`, it's automatically handled
+2. If you are using Webpack or Rollup, you shouldn't transpile `import()` with Babel and let the bundler handle it for you
+3. Otherwise, you need `@babel/plugin-proposal-dynamic-import`
+
+
```

Comment:
@gunn4r Can you rebase on upstream? We have removed versioned_docs so I think only this section is required.